src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -569,6 +570,16 @@ public void selectQuestionNumber(int qnNumber, int newQnNumber) {
         selectDropdownByVisibleValue(qnNumSelect, String.valueOf(newQnNumber));
     }
 
+    public boolean isSelectQuestionNumberEnabled(int qnNumber) {
+        WebElement qnNumSelect = browser.driver.findElement(By.id("questionnum-" + qnNumber));
```

Comment:
Oh sorry! :sweat_smile: I misread your previous comment. Got it now. Will make the necessary changes! :sweat_smile: